### PR TITLE
markdown documentation for multi-cluster functionality

### PR DESCRIPTION
**Pull request by sebastianburckhardt**

### CODE REVIEW

File: src/Documentation/Multi-Cluster/MultiClusterConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+These restrictions ensure that protocols such as the single-instance-protocol can correctly maintain mutual exclusion of activations even under configuration changes.
+
+#### Via Management Grain
+
+Multi-cluster configurations can be injected on any node in any cluster, using the Orleans Management Grain.
+For example, to inject a multi-cluster configuration that consists of the three clusters { A, B, C }, we can pass a string enumerable to the management grain:
+
+```csharp
+   var clusterlist = "A,B,C".Split(',');
```
Comment (jdom) [inline src/Documentation/Multi-Cluster/MultiClusterConfiguration.md]: probably something more relatable is `var clusterList = new[] { "east-us", "west-us", "west-europe" }`

File: src/Documentation/Multi-Cluster/MultiClusterConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+   var mgtGrain = GrainClient.GrainFactory.GetGrain<IManagementGrain>(0);	
+   mgtGrain.InjectMultiClusterConfiguration(clusterlist, "my comment here"));
```
Comment (jdom) [inline src/Documentation/Multi-Cluster/MultiClusterConfiguration.md]: missing `await`

File: src/Documentation/Multi-Cluster/MultiClusterConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+There is an optional third argument, a boolean called `checkForLaggingSilosFirst`, which defaults to true. It means that the system performs a best-effort check to see if there are any silos anywhere that have not caught up to the current configuration yet, and rejects the change if it finds such a silo. This helps to detect violations of the restriction that only one configuration change should be pending at a time (though it cannot guarantee it under all circumstances).
+
+####	Via Default Configuration
+
+In situations where the multi-cluster configuration is known in advance and the deployment is fresh every time (e.g.  for testing), we may want to supply a default configuration. The global configuration supports an optional attribute `DefaultMultiCluster` which takes a comma-separated list of cluster ids:
+
+```html
+   <MultiClusterNetwork ... DefaultMulticluster="A,B,C" ...>
```
Comment (jdom) [inline src/Documentation/Multi-Cluster/MultiClusterConfiguration.md]: should use the more relatable names if you choose to implement my other comment

**Comment by sebastianburckhardt** [comment]
I have edited the names to be more meaningful.

**Comment by jdom** [comment]
Thanks a lot, these docs look really great

**Pull request merged**